module/motion/OldWalkEngine/data/config/igus1/OldWalkEngine.yaml

```diff
@@ -70,6 +70,11 @@ walk_cycle:
   single_support_phase:
     start: 0.3 #0.3
     end: 0.6 #0.6
+  # Second Order Low Pass Filter difference equation:
```

Comment:
If this is the same for every robot then only put it in the top-level config file. Keep robot specific config files for instances where the parameters deviate per robot

Reply:
The config params are intended to be configured for each robot independently. Not sure how uniquely the robots will have to be tuned w.r.t the LPF, but putting them in the config for future use if necessary. Can remove them from robot-specific config files if you think it's better convention.

Reply:
The correct convention is to use the per robot configs only when there are per robot changes (we have been doing this really poorly up until now). But if you think there is a really high likelihood that we will have per robot filtering parameters then only put them in the per robot files and not in the top-level file (I feel like this could be a bad idea though).

Reply:
Okay, will remove